flexx/app/examples/flexx_in_thread.py

```diff
@@ -0,0 +1,51 @@
+"""
+Example showing running Flexx' event loop in another thread.
+This is not a recommended use in general.
+
+Most parts of Flexx are not thread-save. E.g. setting properties
+should generally only be done from a single thread. Event handlers
+are *always* called from the same thread that runs the event loop
+(unless manually called).
+
+The app.create_server() is used to (re)create the server object, binding
+it to the ioloop of the current thread. Note that app.start() must be
+called from that same thread.
+
+"""
+
+import time
+import threading
+
+from flexx import app, event
+
+
+class MyModel1(event.HasEvents):
+    @event.prop
+    def foo(self, v=0):
+        return v
+    
+    @event.connect('foo')
+    def on_foo(self, *events):
+        for ev in events:
+            print('foo changed to', ev.new_value)
+
+# Create model in main thread
+model = MyModel1()
+
+def main():
+    app.create_server()
+    model.foo = 2  # Probably a bad idea to set props from both threads
+    app.start()
+
+# Start Flexx server in new thread
+t = threading.Thread(target=main)
```

Comment:
In this particular case you can, because at that moment there won't already be a server. But it would break if somehow a server is created already. Therefore I think its better to include the call to `app.create_server()`, to make it more rubust/portable.